src/Uno.UI/UI/Xaml/Media/Imaging/SvgImageSource.skia.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.IO;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Windows.UI.Xaml.Media.Imaging;
+
+partial class SvgImageSource
+{
+	private protected override bool TryOpenSourceAsync(CancellationToken ct, int? targetWidth, int? targetHeight, out Task<ImageData> asyncImage)
+	{
+		asyncImage = TryOpenSourceAsync(ct);
+		return true;
+	}
+
+	private async Task<ImageData> TryOpenSourceAsync(CancellationToken ct)
+	{
+		try
+		{
+			if (UriSource != null)
+			{
+				if (UriSource.Scheme == "http" || UriSource.Scheme == "https")
+				{
+					var client = new HttpClient();
```

Comment:
We'll probably need to move this block to a separate file (I think it comes from somewhere else), and reuse the HttpClient (there are potential connection starving issues).

Reply:
Refactored to use a static HttpClient and reuse it. Did similar adjustments in some other parts of the solution